main.go

```diff
@@ -39,7 +39,7 @@ func main() {
 	e.GET("/isgeospatial", handlers.IsGeospatial(appConfig.FileStore))
 	e.GET("/modeltype", handlers.ModelType(appConfig.FileStore))
 	e.GET("/modelversion", handlers.ModelVersion(appConfig.FileStore))
-	e.GET("/geospatialdata", handlers.GeospatialData(appConfig.FileStore))
+	e.GET("/geospatialdata", handlers.GeospatialData(appConfig.FileStore, appConfig.DestinationCRS))
```

Comment:
As mentioned in the comment on geospatialdata.go, you could just pas the appConfig struct here.

Reply:
Done!